src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Data/EmployerAccountRepository.cs

```diff
@@ -69,37 +71,36 @@ public async Task<Accounts<Domain.Models.Account.Account>> GetAccounts(string to
 
         public async Task<AccountDetail> GetAccountDetailByHashedId(string hashedAccountId)
         {
-            AccountDetail accountDetail = null;
-
-            var parameters = new DynamicParameters();
-
-            parameters.Add("@hashedAccountId", hashedAccountId, DbType.String);
-
-            await _db.Value.Database.Connection.QueryAsync<AccountDetail, string, long, AccountDetail>(
-                sql: "[employer_account].[GetAccountDetails_ByHashedId]",
-                param: parameters,
-                transaction: _db.Value.Database.CurrentTransaction.UnderlyingTransaction,
-                commandType: CommandType.StoredProcedure,
-                splitOn: "PayeSchemeId,LegalEntityId",
-                map: (parent, payeSchemeRef, legalEntityId) =>
+            var accountDetail = await _db.Value.Accounts
+                .Where(ac => ac.HashedId == hashedAccountId)
```

Comment:
I know this is how it worked before but we shouldn't be looking up accounts using the hashed ID. I'm happy for this repository method to be deleted and for its body to be moved into the query handler itself if that helps.